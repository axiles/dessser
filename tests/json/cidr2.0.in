[{ "cxgpze": 123, "ydmea": { "v4": { "ip": 1234, "mask": 24 } } }, null]
