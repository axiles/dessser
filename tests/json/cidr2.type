{cxgpze: U40?; hrmehj: U32?; ydmea: Cidr?}?[2]
