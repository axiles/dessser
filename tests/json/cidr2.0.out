((123 null (0 (1234 24))) null)
